badges/lib/rubycritic_badge_score.svg

```diff
@@ -0,0 +1,35 @@
+<svg contentScriptType="text/ecmascript" contentStyleType="text/css" preserveAspectRatio="xMidYMid meet" version="1.0" height="20" width="200"
+  xmlns="http://www.w3.org/2000/svg"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+
```

Comment:
These changes are not described in the commit message! (applies for the ret of the changes in this commit)